_templates/exchanges.html

```diff
@@ -18,6 +18,8 @@ <h2 class="exchanges-tab-title exchanges-international-title" id="international"
     <a class="marketplace-link" href="https://coinmama.com/">Coinmama</a>
     <br>
     <a class="marketplace-link" href="https://www.kraken.com/">Kraken</a>
+    <br>
+    <a class="marketplace-link" href="https://www.rain.bh/">Rain</a>
```

Comment:
As you all are a new exchange, please revise the PR to remove the addition to the International category. We also note that you all do not appear in the list of [250+ exchanges reporting volume](https://coinmarketcap.com/rankings/exchanges/reported/) on CoinMarketCap, which would further preclude the addition to this section.

Reply:
No worries, done. We have our public orderbook API currently in development, so can just open up another PR down the road once we are listed on CMC.

Thanks for your review @wbnns , changes pushed!